appdev/platform/guides/Nexus-5X-Developer-Information.rst

```diff
@@ -0,0 +1,200 @@
+Nexus 5X Developer Information
```

Comment:
Due to Halium becoming the 'correct' way to port Ubuntu Touch, the 5X developer information should not be brought forward.

Reply:
Ok, better to discard it as PR